src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -181,3 +190,21 @@ def make_rate_of_zero(
     key: str, location: str, years: int | str | list[int] | None = None
 ) -> int:
     return 0.0
+
+
+def load_testing_restrictions(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> dict[str, Any]:
+    return get_data(data_keys.ALZHEIMERS.RESTRICTIONS, location, years)
+
+
+def load_testing_prevalence(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> float:
+    return 0.75
```

Comment:
I'm not sure I understand what this even means. What is prevalence in the context of testing?